port/common/omrsysinfo_helpers.c

```diff
@@ -183,7 +183,7 @@ getX86Description(struct OMRPortLibrary *portLibrary, OMRProcessorDesc *desc)
  */
 
 void
-getX86CPUID(uint32_t leaf, uint32_t *cpuInfo)
+OMRgetX86CPUID(uint32_t leaf, uint32_t *cpuInfo)
```

Comment:
I expect the caller of this function needs to be fixed as well.

Reply:
Whoops, I accidentally missed it...

Reply:
Plus, if this is only called within the file, it can be declared static.

Reply:
The purpose of the PR submitted was to migrate these functions from OpenJ9 to OMR. Since the OpenJ9 equivalent functions weren't declared static, then I don't think we should declare these static either.

Reply:
Copying badly written code doesn't seem like a good argument for not improving it.